woocommerce/woocommerce
# Docs/add merchant code snippet docs

### Changes proposed in this Pull Request:

Adds merchants docs snippets to to the code snippets folder in docs. Will work on nesting docs in more salient subcategories at a later date.

### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Make sure the manifest is updated to include all new docs
2. Make sure docs are available in the code snippets menu item
3.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 
This is an addition of docs from the merchant side to the developer documentation based on the existence of code snippets. Should not affect any other files. 
</details>
